src/commands/coin/coin.ts

```diff
@@ -10,7 +17,266 @@ import {
 } from '../../components/coin';
 import { EMBED_COLOUR } from '../../utils/embeds';
 
-@ApplyOptions<SubCommandPluginCommandOptions>({
+// Adjust coin balance
+const coinAdjustExecuteCommand: SapphireMessageExecuteType = async (
+  client,
+  messageFromUser,
+  args
+): Promise<SapphireMessageResponse> => {
+  if (!(<Readonly<Permissions>>messageFromUser.member?.permissions).has('ADMINISTRATOR')) return '';
+
+  // First mandatory argument is user
+  const user = <User>args['user'];
+  if (!user) {
+    throw new Error('please enter a valid user mention or ID for balance adjustment.');
+  }
+
+  // Second mandatory argument is amount
+  const amount = args['amount'];
+  if (!amount) {
+    throw new Error('please enter a valid amount to adjust.');
+  }
+
+  // Optional argument is reason
+  const reason = args['reason'];
+
+  // Adjust coin balance
+  await adjustCoinBalanceByUserId(
+    user.id,
+    <number>amount,
+    UserCoinEvent.AdminCoinAdjust,
+    <string>(reason ? reason : ''),
+    client.user?.id
+  );
+  // Get new balance
+  const newBalance = await getCoinBalanceByUserId(user.id);
+
+  return `${user.username} now has ${newBalance} Codey coins 🪙.`;
+};
+
+const coinAdjustCommandDetails: CodeyCommandDetails = {
+  name: 'adjust',
+  aliases: ['a'],
+  description: 'Adjust the coin balance of a user.',
+  detailedDescription: `**Examples:**
+\`${container.botPrefix}coin adjust @Codey 100\`
+\`${container.botPrefix}coin adjust @Codey -100 Codey broke.\``,
+
+  isCommandResponseEphemeral: true,
```

Comment:
I think this should be false, especially because this command can only be used by administrators. so there should be a visible record of the command usage.

Reply:
I prefer the first implementation, unless others feel more strongly to the second implementation.

Reply:
(meant to reply to the first comment)

Reply:
Sounds good, I've added this